document_quick_access/views/document_quick_access_rule.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2019 Creu Blanca
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="document_quick_access_rule_form_view" model="ir.ui.view">
+        <field
+            name="name"
+        >document.quick.access.rule.form (in document_quick_access)</field>
+        <field name="model">document.quick.access.rule</field>
+        <field name="arch" type="xml">
+            <form>
+                <header>
+                    <button
```

Comment:
The standard is a ribbon, not a button

Reply:
If we use the ribbon then we can't restore it back. so I have used the button.

Reply:
You can, use More Actions